.pre-commit-config.yaml

```diff
@@ -49,5 +49,5 @@ repos:
     rev: 22.3.0
     hooks:
       - id: black
-        language_version: python3.7
```

Comment:
nit: we can probably get rid of this - this should be configured by the default_language_version at the top of this file